metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py

```diff
@@ -1,13 +1,10 @@
 import re
 from textwrap import dedent
-from typing import Any, Dict
-
+from typing import Any, Dict, Optional, List, Tuple
+from datahub.ingestion.source.vertica.common import VerticaSQLAlchemySource , SQLAlchemyConfigVertica 
```

Comment:
Imported Source and Config class from Vertica/common.py file. VerticaSQLAlchemySource class is an extension of BasicSQLAlchemyConfig to support Vertica-related changes.  SQLAlchemyConfigVertica is an extension of SQLAlchemySource to support new functions added in the source to support projections and lineage.